google-codelab-index/google_codelab_index.scss

```diff
@@ -82,22 +88,38 @@ a {
   color: #fff;
   line-height: 40px;
   height: 40px;
-  padding: 0 40px;
+  flex-grow: 1;
+  z-index: 1;
 }
 
-#toolbar #searchbar input::-webkit-input-placeholder {
+#searchbar input::-webkit-input-placeholder {
   color: rgba(255,255,255,.5);
 }
-#toolbar #searchbar input::-moz-placeholder {
+#searchbar input::-moz-placeholder {
   color: rgba(255,255,255,.5);
 }
-#toolbar #searchbar input:-ms-input-placeholder {
+#searchbar input:-ms-input-placeholder {
   color: rgba(255,255,255,.5);
 }
-#toolbar #searchbar input:-moz-placeholder {
+#searchbar input:-moz-placeholder {
```

Comment:
This seems to be almost a duplicate of a few lines above?
Also, I thought firefox didn't need a prefix anymore? https://caniuse.com/#feat=css-placeholder